new_platforms/docs/ta_debugging_wsl.md

```diff
@@ -0,0 +1,108 @@
+Debugging OP-TEE TAs on WSL
+===========================
+
+This document is a complement to [Debugging OP-TEE TAs with
+QEMU](ta_debugging_qemu.md). That guide assumes it is being followed on a native
+Linux system. This guide shows you how to configure the Windows Subsystem for
+Linux (WSL) to use the guide on WSL.
+
+In this guide, you will learn how to:
+
+1. Install and configure a native X Server;
+2. Work around a UNIX feature used by `fakeroot` not yet implemented in WSL.
+
+# Prerequisites
+
+This guide presumes you have a Windows 10 installation, version 1803 or later.
+
+## Required Features and Apps
+
+WSL is not installed by default. To install it, launch PowerShell as
+Administrator and run:
+
+```
+Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+```
+
+Restart your computer when prompted.
+
+You are now ready to install a Linux distribution on WSL. To do so, launch the
+Microsoft Store and search for "Ubuntu 18.04", then click "Get".
```

Comment:
It would be simpler to just link to https://www.microsoft.com/en-us/p/ubuntu-1804-lts/9n9tngvndl3q here

Reply:
Good idea.